optuna/integration/pytorch_distributed.py

```diff
@@ -0,0 +1,281 @@
+from datetime import datetime
+import pickle
+from typing import Any
+from typing import Callable
+from typing import Dict
+from typing import Optional
+from typing import Sequence
+
+import optuna
+from optuna._experimental import experimental
+from optuna._imports import try_import
+from optuna.distributions import BaseDistribution
+from optuna.distributions import CategoricalChoiceType
+
+
+with try_import() as _imports:
+    import torch
+    import torch.distributed as dist
+
+
+@experimental("2.6.0")
+class TorchDistributedTrial(optuna.trial.BaseTrial):
+    """A wrapper of :class:`~optuna.trial.Trial` to incorporate Optuna with PyTorch distributed.
+
+    .. seealso::
+        :class:`~optuna.integration.TorchDistributedTrial` provides the same interface as
+        :class:`~optuna.trial.Trial`. Please refer to :class:`optuna.trial.Trial` for further
+        details.
+
+    See `the example <https://github.com/optuna/optuna/blob/master/
+    examples/pytorch_distributed_simple.py>`__
```

Comment:
Please update here, too.

Reply:
Oops! Thank you for your careful review. I fixed it in https://github.com/optuna/optuna/pull/2303/commits/6976083a8659327629046ae2b994e865be62cd03.